doc/source/api/index.rst

```diff
@@ -83,8 +83,8 @@ API
    .. toctree::
        :maxdepth: 1
 
-       python
```

Comment:
Renamed `python` to `python_bindings` to avoid conflicts with the `python/` directory.